### PR TITLE
feat(replays): Add privacy & legal FAQ, plus additional edits

### PR DESCRIPTION
**Pull request by benvinegar**
Modifies the "Protecting User Privacy" page under Session Replay:
* Adds a Privacy & Legal FAQ
* Adds some new intro paragraphs
* Adds a new heading, "How Session Replay Approaches Data Capture", to the previous content

### CODE REVIEW

File: src/docs/product/session-replay/protecting-user-privacy.mdx [hunk 1]
```diff
@@ -18,12 +27,38 @@ Alternatively, you can turn off the default privacy configuration, which will ca
...
+
+A: Yes, we offer a [Data Processing Addendum (DPA)](https://sentry.io/legal/dpa/) to cover GDPR and CCPA requirements for any personal data sent to Sentry, including via our Session Replay product. You can enter into our DPA by following the instructions [here](https://sentry.io/trust/privacy/).
+
+**Q: I have an existing DPA with Sentry, does it apply to Session Replay?**
+
+A: Yes, see above. The data processing and security commitments in our DPA apply to the personal data you send to the Sentry service (including via our Session Replay product).
+
+**Q: I have an existing Business Associate Addendum (BAA) with Sentry, does it apply to Session Replay?**
+
+A: Yes, [Our BAA](https://sentry.io/legal/baa/) covers HIPAA requirements for any protected health information (PHI) sent to the Sentry service (including if you elect to do so via our Session Replay product).
```
Comment (jas-kas) [inline src/docs/product/session-replay/protecting-user-privacy.mdx:52]:
```suggestion
A: Yes, [our BAA](https://sentry.io/legal/baa/) covers HIPAA requirements for any protected health information (PHI) sent to the Sentry service (including if you elect to do so via our Session Replay product).
```

File: src/docs/product/session-replay/protecting-user-privacy.mdx [hunk 2]
```diff
@@ -18,12 +27,38 @@ Alternatively, you can turn off the default privacy configuration, which will ca
 
 To learn more, including available configuration options, see [Session Replay Privacy](/platforms/javascript/session-replay/privacy/).
 
-## User Input
+### User Input
 
 In addition to HTML content, Session Replay records key user interactions like mouse movements, mouse clicks, and keyboard presses. By default, all user keypresses are redacted and replaced with asterisks. Captured mouse movements are limited to source and target destinations (for example, where the mouse came from and where it went, not the exact path). A low-fidelity approximation of this movement is recreated during playback.
```
Comment (ryan953) [inline src/docs/product/session-replay/protecting-user-privacy.mdx:32]: >... like mouse movements, mouse clicks, and keyboard presses.

Should we clarify that these are keypresses into text-capture areas, ie: inputs with focus? Feels like a technicality but might spark an idea

**Review by jas-kas — APPROVED** [review]

File: src/docs/product/session-replay/protecting-user-privacy.mdx [hunk 3]
```diff
@@ -18,12 +27,38 @@ Alternatively, you can turn off the default privacy configuration, which will ca
...
+
+## Legal and Privacy FAQ
+
+**Q: Can I use Session Replay and comply with the GDPR and CCPA?**
+
+A: Yes, we offer a [Data Processing Addendum (DPA)](https://sentry.io/legal/dpa/) to cover GDPR and CCPA requirements for any personal data sent to Sentry, including via our Session Replay product. You can enter into our DPA by following the instructions [here](https://sentry.io/trust/privacy/).
+
+**Q: I have an existing DPA with Sentry, does it apply to Session Replay?**
+
+A: Yes, see above. The data processing and security commitments in our DPA apply to the personal data you send to the Sentry service (including via our Session Replay product).
```
Comment (ryan953) [inline src/docs/product/session-replay/protecting-user-privacy.mdx:48]: Does the text inside the parenthesis need some extra commas thrown in too?

File: src/docs/product/session-replay/protecting-user-privacy.mdx [hunk 4]
```diff
@@ -18,12 +27,38 @@ Alternatively, you can turn off the default privacy configuration, which will ca
...
+
+**Q: I have an existing Business Associate Addendum (BAA) with Sentry, does it apply to Session Replay?**
+
+A: Yes, [our BAA](https://sentry.io/legal/baa/) covers HIPAA requirements for any protected health information (PHI) sent to the Sentry service (including if you elect to do so via our Session Replay product).
```
Comment (ryan953) [inline src/docs/product/session-replay/protecting-user-privacy.mdx:52]: Does the text inside the parenthesis need some extra commas thrown in too?

File: src/docs/product/session-replay/protecting-user-privacy.mdx (hunk 2 above)
Comment (benvinegar) [reply src/docs/product/session-replay/protecting-user-privacy.mdx:32]: Yes we should, I will follow up.

File: src/docs/product/session-replay/protecting-user-privacy.mdx (hunk 4 above)
Comment (benvinegar) [reply src/docs/product/session-replay/protecting-user-privacy.mdx:52]: Mmm, seems okay to me. Could you elaborate?

**Pull request merged**